instrumentation/github.com/gin-gonic/gin/otelgin/test/gin_test.go

```diff
@@ -518,12 +535,15 @@ func TestMetrics(t *testing.T) {
 
 			attrs := []attribute.KeyValue{
 				attribute.String("http.request.method", "GET"),
-				attribute.Int64("http.response.status_code", 200),
+				attribute.Int64("http.response.status_code", tt.expectStatus),
 				attribute.String("network.protocol.name", "http"),
 				attribute.String("network.protocol.version", fmt.Sprintf("1.%d", r.ProtoMinor)),
 				attribute.String("server.address", "foobar"),
 				attribute.String("url.scheme", "http"),
 			}
+			if tt.expectHasRouteAttr {
```

Comment:
When `expectHasRouteAttr` is `false`, we should also verify that it is not present in the attribute list.

Reply:
I think it's covered in `Attributes: attribute.NewSet(attrs...)` ?

Reply:
You are right, please ignore.